deploy/roles/iptables/tasks/main.yml

```diff
@@ -79,5 +79,11 @@
 - name: Allow outgoing
   command: "iptables -P OUTPUT ACCEPT"
 
+# We need initscripts so we can call /sbin/service in the next task
+- name: Ensure initscripts is installed
```

Comment:
This is RHEL version independent?

Reply:
I'm 99% certain it is, because I've run the `iptables` role dozens of times on both CentOS 6.x and CentOS 7.x.  Do we need more assurance than that?